aten/src/ATen/native/native_functions.yaml

```diff
@@ -2180,6 +2180,14 @@
   use_c10_dispatcher: full
   variants: method  # This is method-only to match the previous tensor API. In the future we could make this a function too.
 
+- func: moveaxis.intlist(Tensor self, int[] src, int[] dst) -> Tensor
+  use_c10_dispatcher: full
+  variants: function
```

Comment:
No method variant?

Reply:
Since `numpy.moveaxis` is function only.

Reply:
It's OK not to have a method variant to start, but just because NumPy doesn't have one doesn't mean we shouldn't, either. The closet analogue to this function is probably [permute](https://pytorch.org/docs/master/tensors.html?highlight=permute#torch.Tensor.permute), which is a method.